pkgs/top-level/all-packages.nix

```diff
@@ -16826,6 +16826,8 @@ with pkgs;
 
   nextpnr = callPackage ../development/compilers/nextpnr { };
 
+  nextpnr-xilinx = callPackage ../development/compilers/nextpnr-xilinx { };
```

Comment:
Please move `pkgs/development/compilers/nextpnr-xilinx/default.nix` to `pkgs/by-name/ne/nextpnr-xilinx/package.nix` and drop this line, thanks!